app/controlplane/api/controlplane/v1/workflow_run.proto

```diff
@@ -30,6 +30,7 @@ service AttestationService {
   rpc GetContract(AttestationServiceGetContractRequest) returns (AttestationServiceGetContractResponse);
   rpc Init(AttestationServiceInitRequest) returns (AttestationServiceInitResponse);
   rpc Store(AttestationServiceStoreRequest) returns (AttestationServiceStoreResponse);
+  rpc StoreBundle(AttestationServiceStoreBundleRequest) returns (AttestationServiceStoreBundleResponse);
```

Comment:
why did you choose to have another endpoint?

Reply:
I wanted to keep them completely independent. Do you suggest using the same endpoint and message? Maybe reusing the `attestation` field for both purposes? That would work as well.

Reply:
or another field in the message called bundle and mark the other one as deprecated?

Reply:
The reason I am thinking about using the same API endpoint it is so we maintain the signature during logging, monitoring, etc

Reply:
Adding another field to the same message won't work because old servers will refuse the request. Check the latest update, I've reused the same field for both purposes. The client will fall back if it finds an old server and send the deprecated payload instead.

Reply:
Nevermind, I've added another field and works as expected.